h/api/test/schemas_test.py

```diff
@@ -42,13 +42,13 @@ def test_it_sets_appropriate_error_message_when_data_invalid(self):
 class TestAnnotationSchema(object):
```

Comment:
Perhaps this is a silly question, but why/since when do we have 350 lines of tests all of which are basically testing `jsonschema`, and not our own code...?